DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml

```diff
@@ -507,96 +104,184 @@ spec:
     - name: git-source
       type: git
   tasks:
-    - name: ace-api-build
+    - name: build-mq
       taskRef:
-        name: build-push-ace-api-docker-image
+        name: build-push-docker-image
       params:
+        - name: imageName
+          value: "mq-ddd"
         - name: imageTag
           value: "$(params.imageTag)"
+        - name: dockerfile
+          value: "MQ.Dockerfile"
+        - name: contextPath
+          value: ""
       resources:
         inputs:
           - name: git-source
             resource: git-source
-    - name: ace-acme-build
+    - name: deploy-mq
+      runAfter:
+        - build-mq
       taskRef:
-        name: build-push-ace-acme-docker-image
+        name: deploy-task
       params:
         - name: imageTag
           value: "$(params.imageTag)"
+        - name: imageName
+          value: "mq-ddd"
+        - name: releaseName
+          value: "mq-ddd-qm-$(params.imageTag)"
```

Comment:
Also, I think this will affect the service name used to access the queue manager, so might need to update the policy in the prereqs to match this.

Reply:
Done